tests/test_cloud_for_customers.py

```diff
@@ -0,0 +1,27 @@
+"""Tests for CloudForCustomers source class"""
+from viadot.config import get_source_credentials as get_credentials
```

Comment:
I would not do that as there is a function with this name in `prefect-viadot`, that might be confusing?

Reply:
Fixed